sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OpenAiController.java

```diff
@@ -120,11 +120,11 @@ Object chatCompletionImage(
     return response.getChoices().get(0).getMessage();
   }
 
-  @GetMapping("/chatCompletionTool")
+  @GetMapping("/chatCompletionToolExecution")
   @Nonnull
-  Object chatCompletionTools(
+  Object chatCompletionToolExecution(
       @Nullable @RequestParam(value = "format", required = false) final String format) {
-    final var response = service.chatCompletionTools(12);
+    final var response = service.chatCompletionToolExecution("Dubai", "°C");
```

Comment:
(Comment)

You know, you could add those default values to the REST endpoint schema... i.e. using `@RequestParam` or `@PathParam`.